libs/map/src/lib/components/layer/wmts-layer/vl-map-wmts-layer.ts

```diff
@@ -92,7 +92,7 @@ export class VlMapWmtsLayer extends VlMapLayer {
         const matrixIds = new Array(16);
         for (let z = 0; z < 16; ++z) {
             resolutions[z] = size / Math.pow(2, z);
-            matrixIds[z] = z;
+            matrixIds[z] = this.__grbMatrixSet +':'+ z;
```

Comment:
O.b.v. attribuut, mogelijks wordt dit niet door elke BE ondersteund.

Reply:
Ik heb twee attributen toegevoegd:
`data-vl-matrix-set` wat default terugvalt op `BPL72VL`.
en `data-vl-matrix-prefix` wat, indien aanwezig, de matrixset zal prefixen.